suite/suite.go

```diff
@@ -199,6 +202,22 @@ func Run(t *testing.T, suite TestingSuite) {
 		}
 		tests = append(tests, test)
 	}
+	if *shuffle != "off" {
```

Comment:
Add an empty line above.

Also add a comment mentionning that the implementation is similar to the implementation of -test.shuffle in package testing linking to https://cs.opensource.google/go/go/+/refs/tags/go1.20.5:src/testing/testing.go;l=1876